web/src/engine/package-cache/src/modelCache.ts

```diff
@@ -11,7 +11,7 @@ export default class ModelManager {
     return this.languageModelMap[lgCode];
   }
 
-  // Accessible publicly as keyman.modelManager.register(model: ModelSpec)
+  // Accessible publicly as keyman.modelCache.register(model: ModelSpec)
```

Comment:
I see in test.js above you changed the `keyman.modelManager.register` reference to `keyman.addModel`, but here you refer to `keyman.modelCache.register`. Which should we be using?

Reply:
I think the comment here was originally written before `addModel` was defined; that's the better one to use & make public, really.